Add single/double quote recommendations (#196)
wintercms/docs: architecture/developer-guide.md

];
```

#### Use of single / double quotes

> Winter CMS highly recommends the use of single quotes around strings instead of double quotes. If such a string contains itself a single quote, we recommend escaping that single quote within the string.

Reviewer: ```suggestion
Winter CMS highly recommends the use of single quotes around strings, instead of double quotes. If the string itself contains a single quote as an apostrophe, you should escape that single quote within the string.
```
